dftimewolf/lib/collectors/bigquery.py

```diff
@@ -25,36 +25,37 @@ def __init__(self,
     self._project_name = ""
     self._query = ""
     self._description = ""
+    self._panda_output = False
 
   # pylint: disable=arguments-differ
-  def SetUp(self, project_name: str, query: str, description: str) -> None:
+  def SetUp(self,
+            project_name: str,
+            query: str,
+            description: str,
+            panda_output: bool) -> None:
     """Sets up a BigQuery collector.
 
     Args:
       project_name (str): name of the project that contains the BigQuery tables.
       query (str): The query to run.
       description (str): A description of the query.
+      panda_output (bool): True if the results should be kep in a panda DF in
+          memory, false if the they should be written to disk.
```

Comment:
kep => kept, panda => pandas, false => False, "if the they" => if they

Reply:
Oh man, that typo density. Done.